js/app.js

```diff
@@ -4,6 +4,15 @@ import Client from "./client"
 import GameState from "./gamestate"
 import GameView from "./gameview"
 
+
+// toggle verbose logging
+let LOG_VERBOSE = true;
```

Comment:
could be "const" instead of "let". also is this actually useful?... if you aren't using LOG_VERBOSE anywhere except 2 lines down, why bother making it a constant.